src/utils/format_date.R

```diff
@@ -0,0 +1,4 @@
+#' Format date in correct form
+format_date <- function(date) {
```

Comment:
i know we figured this out after you wrote the code above, but you can switch it to here (or in an upstream branch):

```
format(date, "%e %B %Y")
```